### PR TITLE
don't set env var prefix name as flask config setting

### PR DESCRIPTION
**Pull request by brassy-endomorph**
Fixes a bug where we were setting unwanted env vars the settings object

### CODE REVIEW

File: aleph/settings.py [hunk 1]
```diff
@@ -5,10 +5,15 @@
 # defaults.
 import json
 import os
-from servicelayer import env
+from datetime import timedelta
```
Comment (Rosencrantz) [inline aleph/settings.py:8]: Is this being used?

File: aleph/settings.py [hunk 2]
```diff
@@ -5,10 +5,15 @@
...
+from json.decoder import JSONDecodeError
 from urllib.parse import urlparse
+
+import structlog
 from flask_babel import lazy_gettext
-from datetime import timedelta
+from servicelayer import env
```
Comment (Rosencrantz) [inline aleph/settings.py:14]: Is this required?

File: aleph/settings.py [hunk 3]
```diff
@@ -154,7 +159,7 @@ def __init__(self) -> None:
         )
 
         # Publishing network diagram embeds
-        self.REACT_FTM_URL = "https://cdn.jsdelivr.net/npm/@alephdata/react-ftm@latest/dist/react-ftm-embed.js"  # noqa
```
Comment (Rosencrantz) [inline aleph/settings.py:157]: Was this a conscious decision?

File: aleph/settings.py (hunk 1 above)
Comment (brassy-endomorph) [reply aleph/settings.py:8]: Yes, for things like

```python
self.ROLE_INACTIVE = timedelta(days=env.to_int("ALEPH_ROLE_INACTIVE", 6 * 30))
```

File: aleph/settings.py (hunk 2 above)
Comment (brassy-endomorph) [reply aleph/settings.py:14]: No not actually, but that was consistent with the style of the file before I first started editing it

File: aleph/settings.py (hunk 3 above)
Comment (brassy-endomorph) [reply aleph/settings.py]: No. `black` must have stripped out the `noqa`. I added it back.

**Pull request merged**